src/old/lib/utilities/API/api.ts

```diff
@@ -311,3 +311,9 @@ export const updatePlatformInformation = async (
 ): Promise<AxiosResponse<PlatformInformationType>> => {
   return instance.put(`/platform-information/`, requestBody);
 };
+
+export const deleteAuthorById = async (
+  id: number,
+): Promise<AxiosResponse<PostsResponseType>> => {
```

Comment:
now I see we have `Promise<AxiosResponse<` everywhere.

but just as information sharing, there is `AxiosPromise<` type which provides exact the same combination

Reply:
Yes, I did as it was in the project.
I didn't know about it. indeed AxiosPromise provides the same behavior. thanks for the info)